scripts/ansible/roles/linux/openenclave/vars/redhat/main.yml

```diff
@@ -30,7 +30,7 @@ yum_packages:
   - "libcurl-devel"
   - "libX11-devel"
   - "ncurses-devel"
-  - "clang"
+  - "clang-8.0.1"
```

Comment:
What version do you get if you don't specify this?

Reply:
9